### PR TITLE
Set up Admin Bundle frontend

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-panel (2.0)
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@sylius-ui/admin",
+  "version": "1.0.0",
+  "license": "MIT",
+  "author": "Sylius Sp. z o.o.",
+  "dependencies": {
+    "@popperjs/core": "^2.11.8",
+    "@symfony/webpack-encore": "^3.1.0",
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/package.json:8]: It's a dev dependency

File: src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import ApexCharts from 'apexcharts';
+
+(function () {
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js:12]: This is vanilla JS "modules" (revealing module pattern), which are not necessary in presence of webpack bundler as this whole file already a CommonJS module (Or some mix of CommonJs/ES modules, I'm not quite sure).

File: src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        color: '#444',
+      },
+    },
+  };
+
+  // eslint-disable-next-line no-undef
+  const ch = document.querySelector('#chart-demo');
+  if (ch) {
+    // eslint-disable-next-line no-undef
+    const chart = new ApexCharts(document.querySelector('#chart-demo'), options);
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js:96]: Shouldn't this part of attaching chart to element be at `app.js` as it was in previous version?

File: src/Sylius/Bundle/AdminBundle/index.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    adminConfig.externals = { ...adminConfig.externals, window: 'window', document: 'document' };
+    adminConfig.name = 'admin';
+
+    Encore.reset();
+
+    return adminConfig;
+  }
+}
+
+module.exports = SyliusAdmin;
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/index.js:44]: I'm not sure about best syntax, but you might as well write as `module.exports.getWebpackConfig()`, I think intermediate object(as container) is not necessary.

File: src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+ * file that was distributed with this source code.
+ */
+
+import ApexCharts from 'apexcharts';
+
+(function () {
+  const options = {
+    series: [{
+      name: 'Inflation',
+      data: [2.3, 3.1, 4.0, 10.1, 4.0, 3.6, 3.2, 2.3, 1.4, 0.8, 0.5, 0.2],
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js:16]: Test data and untranslatable months are alight for test, but I assume this file won't be committed as is.

File: src/Sylius/Bundle/AdminBundle/package.json (hunk 1 above)
Comment (jakubtobiasz) [reply src/Sylius/Bundle/AdminBundle/package.json:8]: From our perspective it's a regular dependency. `@sylius-ui/admin` is a dev dependency for end-projects.

**Pull request merged**